### PR TITLE
Add UI test to verify venue's tips are being handled properly.

### PR DESCRIPTION
**Pull request by untalfranfernandez**
I've created this PR against 'uitest-venudetail' instead of 'master' because we could consider this the second part of that job.

### CODE REVIEW

File: FoursquareTop/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -22,3 +22,6 @@
 "VenueDetail.Status.AccessibilityLabel" = "The venue's status";
 "VenueDetail.Rating.AccessibilityLabel" = "The venue's rating";
 "VenueDetail.Pricing.AccessibilityLabel" = "The venue's pricing tag";
+
+"VenueDetail.Tips.AccessibilityLabel" = "The venue's tips";
```
Comment (gragera) [inline FoursquareTop/Base.lproj/Localizable.strings]: Please add English and Spanish translations

File: FoursquareTopTests/VenueDetailViewControllerDriver.swift [hunk 2]
```diff
@@ -63,8 +63,31 @@ class VenueDetailViewControllerDriver {
...
+        for tip in tips {
+            expectView(withText: tip, toBeVisible: true)
+        }
+    }
+    
+    // MARK: Private
+    
     private func expectView(withText text: String? = nil, withAccessibilityLabel label: String? = nil, toBeVisible visible: Bool) {
         
+        guard text != nil || label != nil else {
```
Comment (gragera) [inline FoursquareTopTests/VenueDetailViewControllerDriver.swift]: I added a comment in the other PR regarding exactly this xD

**Review by gragera — APPROVED** [review]
LGTM, check the comments though

File: FoursquareTop/Base.lproj/Localizable.strings (hunk 1 above)
Comment (fjfdeztoro) [inline FoursquareTop/Base.lproj/Localizable.strings]: 👍

**Review by davideme — APPROVED** [review]

**Pull request closed without merging**